### PR TITLE
Don't show the "WooCommerce Setup" widget in dashboard if WC Admin is disabled

### LINKED ISSUE #29567: wc_admin_url called from dashboard setup
**Describe the bug**
Error: Call to undefined function wc_admin_url() in `woocommerce/includes/admin/class-wc-admin-dashboard-setup.php` line 116

**Expected behavior**
Do not call functions defined within wc-admin package only from the core or at least add a function_exists wrapper.

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Disable wc-admin via filter
2. Open WP-Admin dashboard

### PR DESCRIPTION
**Pull request by Konamiman**
### All Submissions:

* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

When the WooCommerce Admin component is disabled it doesn't make much sense to display the "WooCommerce Setup" widget, as clicking its "Start selling" button will produce an error; additionally, the entire dashboard will crash if the package is disabled (e.g. via the `woocommerce_admin_disabled` filter). So this pull request hides the widget when the package is disabled.

Closes #29567.

### How to test the changes in this Pull Request:

1. Make sure that you see the "WooCommerce Setup" widget in the dashboard (if you don't, delete the `woocommerce_task_list_complete` and `woocommerce_task_list_hidden` options).
2. Disable the admin package programmatically with `add_filter( 'woocommerce_admin_disabled', '__return_true' );` and reload the dashboard.

Without this fix (and in WooCommerce 5.2 RC 2) you'll get a "Call to undefined function wc_admin_url()" error, with this fix the dashboard will load and will not display the "WooCommerce Setup" widget.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Fix - Error when loading the admin dashboard while the admin package was disabled.

### CODE REVIEW

**Review by roykho — DISMISSED** [review]
Tested well. However I do see another method that also have this check here https://github.com/woocommerce/woocommerce/blob/trunk/includes/admin/class-wc-admin-dashboard.php#L65

But this code shouldn't even be fired if WC Admin isn't enabled. But I am just pointing it out in case we should be extra cautious. I don't have any recommendation here but just to be complete. I will leave that up to your discretion. Approving this for now. Feel free to merge this once you decide if you want to apply similar fix in the other area.

**Comment by Konamiman** [comment]
@roykho Indeed, for completeness I've added an extra check in the method you pointed out. Another review plz.

**Review by roykho — APPROVED** [review]
LGTM!

**Pull request merged**
